arviz/plots/autocorrplot.py

```diff
@@ -29,48 +29,48 @@ def plot_autocorr(
 
     Parameters
     ----------
-    data: obj
+    data : InferenceData
         Any object that can be converted to an :class:`arviz.InferenceData` object
         refer to documentation of :func:`arviz.convert_to_dataset` for details
-    var_names: list of variable names, optional
+    var_names : list of str, optional
         Variables to be plotted, if None all variables are plotted. Prefix the
         variables by ``~`` when you want to exclude them from the plot. Vector-value
         stochastics are handled automatically.
-    filter_vars: {None, "like", "regex"}, optional, default=None
+    filter_vars : {None, "like", "regex"},  default=None
         If `None` (default), interpret var_names as the real variables names. If "like",
         interpret var_names as substrings of the real variables names. If "regex",
         interpret var_names as regular expressions on the real variables names. A la
         ``pandas.filter``.
-    max_lag: int, optional
+    max_lag : int, optional
         Maximum lag to calculate autocorrelation. Defaults to 100 or num draws,
         whichever is smaller.
-    combined: bool, default=False
+    combined : bool, default=False
         Flag for combining multiple chains into a single chain. If False, chains will be
         plotted separately.
     grid : tuple
         Number of rows and columns. Defaults to None, the rows and columns are
         automatically inferred.
-    figsize: tuple
+    figsize : tuple 
```

Comment:
to follow matplotlib's documentation (where figsize is taken/inspired from) I would use update this to:

```suggestion
    figsize : (float, float), optional
```

Ref: https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.figure.html (use the same type they use but with `optional` instead of the rcParams default.